lib/reporting/identity_verification_report.rb

```diff
@@ -53,6 +53,17 @@ module Results
       # rubocop:enable Layout/LineLength
     end
 
+    # Because historically fraud-related events were not tagged with SP data,
+    # we need pull these out-of-band events *even if* the are marked as
+    # pending fraud review. This allows us to attribute untagged fraud-related
+    # events (by matching on user_id). We filter these events for counting
+    # purposes, though.
+    EVENTS_TO_IGNORE_IF_FRAUD_REVIEW_PENDING = [
```

Comment:
😍 the pattern of being really explicit in naming things like this, as a good way to prevent future bugs. 🥇